src/rgw/rgw_main.cc

```diff
@@ -1153,6 +1164,10 @@ int main(int argc, const char **argv)
   register_async_signal_handler(SIGUSR1, handle_sigterm);
   sighandler_alrm = signal(SIGALRM, godown_alarm);
 
+  if ( rgw_rate_limit_init() < 0 ) {
+    dout(0) << "Failed to init rate limiter" << dendl;
```

Comment:
Agreed, and I was thinking of letting traffic through while rate-limiting is being initially tested; and add a later change to fail startup.
But then, we want to find & fix bugs early, so will change this ...